README.md

```diff
@@ -1,45 +1,53 @@
 # Gravity Web UI packages monorepo
+
 !["Gravity UI Web monorepo" banner image](./docs/gravity-ui-web-repo-img.png)
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/buildit/gravity-ui-web.svg)](https://greenkeeper.io/)
 [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
 
-[Lerna](https://lernajs.io/) monorepo containing **Gravity design system** NPM packages relating to Web UIs:
-
-* [Gravity UI Web](https://github.com/buildit/gravity-ui-web/tree/develop/packages/gravity-ui-web): The core SASS/CSS library used by all Gravity-powered web projects.
-* [Gravity UI Nunjucks](https://github.com/buildit/gravity-ui-web/tree/develop/packages/gravity-ui-web): Nunjucks HTML templates for Gravity UI components. Also used to generate our [living pattern library](http://style.buildit.digital/).
-* Custom Semantic Release helpers:
-  * [Lerna Config](https://github.com/buildit/gravity-ui-web/tree/develop/packages/semantic-release/config)
-  * [Lerna Analyzer](https://github.com/buildit/gravity-ui-web/tree/develop/packages/semantic-release/lerna-analyzer)
-  * [Lerna Release Notes Generator](https://github.com/buildit/gravity-ui-web/tree/develop/packages/semantic-release/release-notes-generator)
+[Lerna](https://lerna.js.org/) monorepo containing **Gravity design system** NPM packages relating to Web UIs:
 
+- [Gravity UI Web](https://github.com/buildit/gravity-ui-web/tree/develop/packages/gravity-ui-web): The core SASS/CSS library used by all Gravity-powered web projects.
+- [Gravity UI Nunjucks](https://github.com/buildit/gravity-ui-web/tree/develop/packages/gravity-ui-web): Nunjucks HTML templates for Gravity UI components. Also used to generate our [living pattern library](http://style.buildit.digital/).
+- Custom Semantic Release helpers:
+  - [Lerna Config](https://github.com/buildit/gravity-ui-web/tree/develop/packages/semantic-release/config)
+  - [Lerna Analyzer](https://github.com/buildit/gravity-ui-web/tree/develop/packages/semantic-release/lerna-analyzer)
+  - [Lerna Release Notes Generator](https://github.com/buildit/gravity-ui-web/tree/develop/packages/semantic-release/release-notes-generator)
 
 ## Development
 
 ### Initial setup
+
 After an initial clone, or if any of the packages' dependencies have changed you should run the following **in the repo's root directory**:
 
 ```
 npm install
-npm lerna:bootstrap
+lerna bootstrap
 ```
 
 This will install dependencies for all packages in the repo and configure any local cross-references between them.
 
+You may need to install [Lerna](https://lerna.js.org/) globally using:
+
+```
+npm lerna:bootstrap
```

Comment:
If you revert the bit above, you can get rid of this too.

FWIW, this would just run `lerna bootstrap` - but using the copy of lerna installed into Gravity's `node_modules`. To install lerna globally, you'd need to do `npm install -g lerna`.

Reply:
👍